Prac07/guitarTest.py

```diff
@@ -0,0 +1,31 @@
+from Prac07.guitar import Guitar
+
+
+def main():
+
+    i = 0
+    guitars = []
+    while True:
+        name = input("Name: ")
+        if not name:
+            break
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_object = Guitar(name, year, cost)
+        guitars.append(guitar_object)
+        print(guitar_object)
```

Comment:
For consistency, it would be ideal to get the {,.2f} for this print as well, can be done in the classes by making a method there, just like the __str__ method, and calling it here.